lib/static/static/js/controllers/repositoriesCtrl.js

```diff
@@ -1,29 +1,67 @@
 (function() {
-  angular.module('huburn').controller('repositoriesCtrl', ['$scope', '$http', 'navigationService', function($scope, $http, navigationService) {
-    $scope.repositories = [];
+  angular
+    .module('huburn')
+    .controller('repositoriesCtrl', repositoriesCtrl);
+
+  repositoriesCtrl.$inject = ['$scope', '$http', 'navigationService'];
 
+  function repositoriesCtrl($scope, $http, navigationService) {
+    
+    $scope.repositories = [];
     navigationService.clear();
 
-    $http.get('/github', { params: { path: '/user/repos' } }).success(function(repos) {
-      addRepos(repos);
-    });
+    var addUserRepos = function() {
+      $http.get('/github', { params: { path: '/user/repos' } }).success(function(repos) {
+        repos.forEach(function(repo) {
+          addRepoWithAnyMilestoneThatHasAssociatedIssues(repo);
+        });
+      });
+    };
 
-    $http.get('/github', { params: { path: '/user/orgs' } }).success(function(orgs) {
-      orgs.forEach(function(org) {
-        $http.get('/github', { params: { path: '/orgs/' + org.login + '/repos' } }).success(function(repos) {
-          addRepos(repos);
+    var addOrgRepos = function() {
+      $http.get('/github', { params: { path: '/user/orgs' } }).success(function(orgs) {
+        orgs.forEach(function(org) {
+          addFilteredOrgRepos(org);
         });
       });
-    });
+    };
 
-    var addRepos = function(repos) {
-      var newRepos = $scope.repositories.concat(repos);
+    var addFilteredOrgRepos = function(org) {
+      $http.get('/github', { params: { path: '/orgs/' + org.login + '/repos' } }).success(function(repos) {
+        repos.forEach(function(repo) { 
+          addRepoWithAnyMilestoneThatHasAssociatedIssues(repo);
+        });
+      });
+    };
+
+    var addRepoWithAnyMilestoneThatHasAssociatedIssues = function(repo) {
+      var milestonesPath = '/repos/' + repo.owner.login + '/' + repo.name + '/milestones';
+      $http.get('/github', { params: { path: milestonesPath } }).success(function(milestones) {
+        if (anyMilestoneHasAssociatedIssues(milestones))
+          addRepo(repo);
+      });
+    }
+
+    var anyMilestoneHasAssociatedIssues = function(milestones) {
+      return milestones.some(function(milestone) {
+        if (milestone.open_issues > 0 || milestone.closed_issues > 0)
```

Comment:
isn't this the same as "return (milestone.open_issues > 0 || milestone.closed_issues > 0);" ?  Also, good find with the ".some" function.  Do we have to check both open and closed, or can we just see something like "issues" or "total_issues"?